subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -89,7 +91,7 @@ validation_info_find(u32_t start_address, u32_t search_distance)
 
 
 static bool validate_firmware(u32_t fw_dst_address, u32_t fw_src_address,
-		const struct fw_firmware_info *fw_info, bool external)
+		const struct fw_info *fw_info, bool external)
```

Comment:
Is it problematic that the variable has the same name as the type? Genuine question, not pointing in any direction :)

Reply:
A bit confusing at least :P. I'll try to change it.